bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml

```diff
@@ -359,6 +384,13 @@ limitations under the License.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <configuration>
+                    <usedDependencies>org.apache.hbase:hbase-testing-util,org.apache.hbase:hbase-server</usedDependencies>
```

Comment:
please add a comment that this necessary to force hbase version in the common tests?